test/java/seedu/addressbook/common/UtilsTest.java

```diff
@@ -0,0 +1,40 @@
+package seedu.addressbook.common;
+
+import static org.junit.Assert.*;
+
+import java.util.ArrayList;
+
+import org.junit.Test;
+
+public class UtilsTest {
+
+    @Test
+    public void isNoAnyNull() {
+        assertFalse(Utils.isAnyNull("qwerty", 98765431, "guy@girl.com"));
+    }
+
+    @Test
+    public void isYesAnyNull() {
+        assertTrue(Utils.isAnyNull("qwerty", 98765431, null, "guy@girl.com"));
+    }
+
+    @Test
+    public void elementsAreNotUnique() {
+        ArrayList<Object> testObjects = new ArrayList<Object>();
+        testObjects.add(new String("qwerty"));
+        testObjects.add(new Integer(987654321));
+        testObjects.add(new String("guy@girl.com"));
+        testObjects.add(new String("qwerty"));
+        assertFalse(Utils.elementsAreUnique(testObjects));
+    }
+
+    @Test
+    public void elementsAreYesUnique() {
+        ArrayList<Object> testObjects = new ArrayList<Object>();
+        testObjects.add(new String("qwerty"));
```

Comment:
going further you can consider moving all the steps necessary for the test to run (or preconditions) to a set up method before beginning the tests; correspondingly can put all the tear down in a method after the tests.